scripts/import-data.ts

```diff
@@ -70,12 +70,35 @@ async function main(): Promise<void> {
       );
     }
 
-    // Transform data format
+    // Transform data format with user support
     console.log('转换数据格式...');
     const organizations = limitedData
       .map((row) => {
         try {
-          return DataTransformer.transformOrganization(row);
+          const organization = DataTransformer.transformOrganization(row);
+
+          // Extract user data from the same row
+          const userData = DataTransformer.transformUser(row);
+
+          // Attach original data and user info as non-enumerable properties
+          Object.defineProperty(organization, '_originalData', {
+            value: row,
+            enumerable: false,
+            writable: false,
+            configurable: false,
+          });
+
+          // Attach user data for later processing
+          if (userData) {
+            Object.defineProperty(organization, '_userData', {
+              value: userData,
+              enumerable: false,
+              writable: false,
+              configurable: false,
+            });
+          }
```

Comment:
```suggestion
          if (userData)
            userWeakMap.set(organization, userData);
```
`WeakMap` 最适合运行时对象间关联，无痕、自动垃圾回收。